README.md

```diff
@@ -90,6 +90,27 @@
 
    * PULL_SECRET:                   A cluster pull secret retrieved from [Red Hat OpenShift Cluster Manager](https://cloud.redhat.com/openshift/install/azure/installer-provisioned)
 
+   Credentials explained:
+   `AZURE_CLUSTER_CLIENT_*` - Cluster credentials, provided by user and used by
+   cluster components itself. If provided with access `Application.ReadWrite.OwnedBy`,
```

Comment:
Actually this isn't true.  If it has Application.ReadWrite.OwnedBy, we break.  Currently we only grant the SP Contributor on the RG, so it couldn't grant newly minted SPs permissions on the RG even if it wanted to.  We need a validation to ensure the passed SP *doesn't* have that permission.

Reply:
fixed